seed/seedtest/seedtest.go

```diff
@@ -160,9 +167,13 @@ func (s *TestingSeed) MakeModelAssertionChain(brandID, model string, extras ...m
 	return assertChain
 }
 
-func (s *TestingSeed) WriteAssertions(fn string, assertions ...asserts.Assertion) {
-	multifn := filepath.Join(s.AssertsDir, fn)
-	f, err := os.OpenFile(multifn, os.O_CREATE|os.O_WRONLY, 0644)
+func (s *TestingSeed16) WriteAssertions(fn string, assertions ...asserts.Assertion) {
```

Comment:
this function is effectively duplicated with the one in the helpers test, but in the words of go proverbs:

> A little copying is better than a little dependency